index.html

```diff
@@ -15,7 +15,135 @@
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
-  <body>
+
+  <body id="all">
+    
+
+   <div class="head">
+    <nav class="navbar">
+    <ul class="nav">
+      <li>
+      <a href="#all">Home</a>
+    </li>
+    <li>
+      <a href="#articles">Articles</a>
+    </li>
+    <li>
+      <a href="#po">Popular-posts</a>
+    </li>
+    </ul>
+    </nav>
+
+
+
+
+    <div class="title">
+    <h1 class="elirose">EllieRose Acupuncture Blog</h1>
+    <p class="eli">Welcome to my Blog</p>
+    </div>
+   </div>
+ <main class="flex">
+  <div class="main" id="articles"> 
+    <div class="part1">
+      <div class="acu">
+        <img class="acu-img" src="/images/pexels-antoni-shkraba-6076146.jpg">
+        <div class="">
+          <h2>Acupuncture History</h2>
+          <h4>Past, Present, and Future</h4>
```

Comment:
Generally, not a good idea to skip heading levels, i.e. h2->h4, skipping h3. And do not use h1,h2,h3 etc for font size or style. h1,h2,h3 are for meaning: h1 is the main header, h2 is a subheader, h3 is a subheader within h2. If you want to adjust the font size or font style, use css